### PR TITLE
Include fixed tests

### PR DESCRIPTION
**Pull request by blegat**
The other ones should be fixed in:

- [x] https://github.com/jump-dev/MathOptInterface.jl/pull/2555
- [x] https://github.com/jump-dev/MathOptInterface.jl/pull/2556

### CODE REVIEW

**Comment by blegat** [comment]
They have been merged, just need to wait for a new MOI release before we can merge this

**Review by DimitriAlston — APPROVED** [review]

**Pull request merged**
